Source/Scene/Cesium3DTile.js

```diff
@@ -206,14 +231,17 @@ define([
         },
 
         /**
-         * Get the tight oriented bounding box
+         * Get the bounding volume of the tile
          *
-         * @type {Promise}
+         * @type {Object}
          * @readonly
          */
-        orientedBoundingBox : {
+        boundingVolume : {
```

Comment:
It's used [here](https://github.com/e-andersson/cesium/blob/3d-tiles_bounding_volumes/Source/Scene/Instanced3DModel3DTileContentProvider.js#L49) (which is later passed on to `ModelInstanceCollection`) and [here](https://github.com/e-andersson/cesium/blob/3d-tiles_bounding_volumes/Source/Scene/Points3DTileContentProvider.js#L58-L59). I think the name change is fine even if it returns the tile's bounding volume.